src/java/org/apache/cassandra/db/ExpirationDateOverflowHandling.java

```diff
@@ -62,23 +62,23 @@
     }
 
     public static final String MAXIMUM_EXPIRATION_DATE_EXCEEDED_WARNING = "Request on table {}.{} with {}ttl of {} seconds exceeds maximum supported expiration " +
-                                                                          "date of 2038-01-19T03:14:06+00:00 and will have its expiration capped to that date. " +
+                                                                          "date of 2106-02-07T06:28:14+00:00 and will have its expiration capped to that date. " +
                                                                           "In order to avoid this use a lower TTL or upgrade to a version where this limitation " +
-                                                                          "is fixed. See CASSANDRA-14092 for more details.";
+                                                                          "is fixed. See CASSANDRA-14092 and CASSANDRA-14227 for more details.";
 
     public static final String MAXIMUM_EXPIRATION_DATE_EXCEEDED_REJECT_MESSAGE = "Request on table %s.%s with %sttl of %d seconds exceeds maximum supported expiration " +
-                                                                                 "date of 2038-01-19T03:14:06+00:00. In order to avoid this use a lower TTL, change " +
+                                                                                 "date of 2106-02-07T06:28:14+00:00. In order to avoid this use a lower TTL, change " +
```

Comment:
Also `2106-02-07T06:28:13+00:00` if we want that sentinel value for some undetermined future use.